client/coral-embed-stream/src/tabs/stream/components/DraftArea.js

```diff
@@ -35,22 +38,30 @@ export default class DraftArea extends React.Component {
       charCountEnable,
       maxCharCount,
       onChange,
+      comment,
     } = this.props;
 
+    const tASettings = {
+      value,
+      placeholder,
+      id,
+      onChange,
+      rows,
+      disabled,
+      comment,
```

Comment:
Don't pass comment here as a regular property, it should be passed as `queryData`, and a `DraftArea` Container with the slot fragments must be created and several other containers connecting up to the  `Comment` fragment.

Reply:
Otherwise we have interdependent data dependencies of different components which we want to avoid. This also wouldn't work in relay because they have an accurate props filter.